ckan/logic/action/get.py

```diff
@@ -2414,14 +2414,18 @@ def status_show(context, data_dict):
     :rtype: dictionary
 
     '''
+
+    plugins = config.get('ckan.plugins') 
+    extensions = "" if plugins == None else plugins.split()
```

Comment:
```suggestion
    extensions = plugins.split() if plugins else [] 
```
we should always store a list